cmd/harborAPI.go

```diff
@@ -129,6 +130,27 @@ func update(token string, url string, data interface{}) {
 	}
 }
 
+// GetLogs returns all logs for a shipment
+// TODO: add a tail flag feature (jkurz)
```

Comment:
Ideally, we should try to mimic the [docker-compose `logs` command](https://docs.docker.com/compose/reference/logs/).